src/Command/DetectCommand.php

```diff
@@ -48,15 +47,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($application === null) {
             $io->error('Could not find application from command.');
 
-            return self::FAILURE;
+            return 1;
```

Comment:
I prefer we add a `private const FAILURE = 1;` in these command classes and keep using the constant.

Reply:
Done
I just needed to put constant as `public` for keeping compatibility with Symfony Console 5